internal/beater/beater.go

```diff
@@ -844,7 +844,11 @@ func (s *Runner) newLibbeatFinalBatchProcessor(
 		output, err := outputs.Load(indexSupporter, beatInfo, stats, outputName, s.outputConfig.Config())
 		return outputName, output, err
 	}
-	pipeline, err := pipeline.Load(beatInfo, monitors, pipeline.Config{}, nopProcessingSupporter{}, outputFactory)
+	var pipelineConfig pipeline.Config
```

Comment:
As a follow-up probably, I wonder if it would be possible to have a test to assert that the full pipeline config is loaded correctly.

Reply:
Should be possible, but I didn't have the patience to figure it out right now - the system test should cover us for the moment. I've opened https://github.com/elastic/apm-server/issues/11535